runtime/src/main/java/org/corfudb/runtime/exceptions/TransactionAbortedException.java

```diff
@@ -2,6 +2,7 @@
 
 import lombok.Getter;
 import org.corfudb.protocols.wireprotocol.TxResolutionInfo;
```

Comment:
well, that's a bigger change. I don't see a reason to hold off on this PR for that reason.